### PR TITLE
simplify examples structure

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Simplify the example structure and make it more shallow...
This is a reaction to creating `lightning-bolts` so here we keep just the minimal example set to illustrate the Lightning potential :]

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pl_examples/README.md [hunk 1]
```diff
@@ -1,11 +1,67 @@
...
+To run this demo do the following:
+
+1. Log into the jumphost node of your SLURM-managed cluster.  
+2. Create a conda environment with Lightning and a GPU PyTorch version.   
+3. Choose a script to submit    
+
+#### DDP  
+Submit this job to run with distributedDataParallel (2 nodes, 2 gpus each)
+```bash
+sbatch ddp_job_submit.sh YourEnv
```
Comment (jeffling) [inline pl_examples/README.md]: It looks like these files were moved to be hidden. Do we need to change the sbatch call as well?

Comment (Borda) [reply pl_examples/README.md]: Well it may not the desired thing to be hidden, have not realized on Linux this potential issue, will change it...

**Comment by williamFalcon** [comment]
@Borda can we close or merge this please?

**Comment by Borda** [comment]
@williamFalcon merge pls...

**Comment by williamFalcon** [comment]
@Borda waiting on GPU tests here

**Comment by Borda** [comment]
the GPU test will not pass because the master is broken... can you run it locally?

**Comment by williamFalcon** [comment]
@Borda rebase?

**Comment by Borda** [comment]
@williamFalcon rebased and waiting for tests to pass

**Comment by Borda** [comment]
@williamFalcon done :]

**Pull request merged**